bench_manager/bench_manager/doctype/site/site.py

```diff
@@ -216,9 +216,10 @@ def get_installable_apps(doctype, docname):
 @frappe.whitelist()
 def get_removable_apps(doctype, docname):
 	verify_whitelisted_call()
-	removable_apps = frappe.get_doc(doctype, docname).app_list.split("\n")
-	removable_apps.remove("frappe")
-	return removable_apps
+	installed_apps = frappe.get_doc(doctype, docname).app_list.split("\n")
```

Comment:
Where are we using this?

Reply:
this return list for pop 
list contains apps installed on the site, and the user can uninstall one.